JSDemos/Demos/PivotGrid/SummaryDisplayModes/description.md

```diff
@@ -1 +1,32 @@
-The PivotGrid component supports several modes for summary calculation, such as&nbsp;a&nbsp;variation or&nbsp;percent of&nbsp;a&nbsp;total value. Select the required mode from the context menu appearing when you right-click the &laquo;Relative Sales&raquo; field.
\ No newline at end of file
+Summary display modes are post-processing functions that allow you perform additional calculations on each summary value and take into account neighboring summary values. You can use summary display modes to present pivot data from different angles.
```

Comment:
```
are post-processing functions
``` 
Can we give a link to a descriptive topic here? I see there is the related one:  
https://js.devexpress.com/Documentation/Guide/UI_Components/PivotGrid/Summaries/#Summary_Post-Processing
Is it worth it?


```
that allow you perform
```
May be "that allow you TO perform"?